types/unordered_set/fundamental/read.C

```diff
@@ -0,0 +1,68 @@
+#include <ROOT/REntry.hxx>
+#include <ROOT/RNTupleReader.hxx>
+
+using ROOT::Experimental::REntry;
+using ROOT::Experimental::RNTupleReader;
+
+#include <cstdint>
+#include <fstream>
+#include <ostream>
+#include <string>
+#include <string_view>
+#include <unordered_set>
+
+using UnorderedSet = std::unordered_set<std::int32_t>;
+
+static void PrintUnorderedSetValue(const REntry &entry, std::string_view name,
+                                   std::ostream &os, bool last = false) {
+  UnorderedSet &value = *entry.GetPtr<UnorderedSet>(name);
+  os << "    \"" << name << "\": [";
+  bool first = true;
+  for (auto element : value) {
```

Comment:
Is the order in an unordered set 'deterministic' or more exactly the 'same' on all platforms?

Reply:
In my understanding, it's not for the `unordered` containers. I proposed that we sort the read values: https://github.com/root-project/rntuple-validation/pull/39#discussion_r1978883514